3-enrich/emr-etl-runner/build.sh

```diff
@@ -15,6 +15,8 @@
 # Copyright:: Copyright (c) 2015 Snowplow Analytics Ltd
 # License::   Apache License Version 2.0
 
+set -e
```

Comment:
Not sure if it's applicable, but `set -e` "is considered harmful" :-) 

See http://mywiki.wooledge.org/BashFAQ/105

Reply:
So, article advises to avoid `set -e` because some apps intentionally return non-zero as success. However, I'm totally sure this isn't the case of any below commands and what is more important - absence of `set -e` can bite much harder - asset with failed tests will be silently published. But thanks for article, I never actually realized `set -e` is so dumb.